<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Updater</class>
 <widget class="QWidget" name="Updater">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>294</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>294</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>550</width>
    <height>294</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FrameGUI - Updates</string>
  </property>
  <widget class="QLabel" name="TitleLabel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>7</y>
     <width>150</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt; font-weight:700;&quot;&gt;It's Time to Update!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="ToDoLabel">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>264</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose an action:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="UpdateBttn">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>259</y>
     <width>95</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>32</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>32</height>
    </size>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Update Now</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RemindBttn">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>259</y>
     <width>120</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>32</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>32</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Remind Me Later</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SkipBttn">
   <property name="geometry">
    <rect>
     <x>425</x>
     <y>259</y>
     <width>118</width>
     <height>32</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>32</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>32</height>
    </size>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="text">
    <string>Skip This Update</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>63</y>
     <width>536</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="DetailsLabel">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>38</y>
     <width>360</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Below are the details for the next new update for EncodeGUI</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
